.github/workflows/pre-compile_llvm_windows.yml

```diff
@@ -0,0 +1,49 @@
+name: Pre-compile llvm for Windows
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'release_*x'
+
+jobs:
+  build-x64:
+    name: Windows x64
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Windows
+        uses: llvm/actions/setup-windows@main
+        with:
+          arch: amd64
+
+      - name: Setup Ninja
+        run: choco install ninja
```

Comment:
`.github/workflows/llvm-project-tests.yml` uses the following; consider sticking to it:
```yaml
uses: llvm/actions/install-ninja@main
```